qgsquick/from_qgis/plugin/editor/qgsquickvaluerelation.qml

```diff
@@ -186,11 +186,11 @@ Item {
       Image {
         id: rightArrow
         source: QgsQuick.Utils.getThemeIcon("ic_angle_right")
-        width: 15
+        height: parent.height * 0.4
+        width: height / 2
         anchors.right: parent.right
-        anchors.rightMargin: 10
+        anchors.rightMargin: 15
```

Comment:
The same applies here for icon as comments on valuemap icon. Also would be nice to have  color overlay for both icons